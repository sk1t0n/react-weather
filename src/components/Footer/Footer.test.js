import React from 'react';
import renderer from 'react-test-renderer';
import { Footer } from './index';

test('renders correctly footer component', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
