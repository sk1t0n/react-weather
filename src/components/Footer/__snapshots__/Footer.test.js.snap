// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly footer component 1`] = `
<footer
  className="footer py-3 bg-dark"
>
  <div
    className="container"
  >
    <span>
      Copyright © 
      2021
    </span>
    <span>
        |  All rights reserved  |  
    </span>
    <a
      className="link"
      href="https://github.com/sk1t0n/react-weather"
    >
      Anton Grabovsky
    </a>
  </div>
</footer>
`;

exports[`renders correctly footer component 1`] = `
<footer
  className="footer py-3 bg-dark"
>
  <div
    className="container"
  >
    <span>
      Copyright © 
      2021
    </span>
    <span>
        |  All rights reserved  |  
    </span>
    <a
      className="link"
      href="https://github.com/sk1t0n/react-weather"
    >
      Anton Grabovsky
    </a>
  </div>
</footer>
`;
